javaagent-api/src/main/java/io/opentelemetry/javaagent/instrumentation/api/db/CassandraQueryNormalizer.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.instrumentation.api.db;
+
+import io.opentelemetry.javaagent.instrumentation.api.db.normalizer.ParseException;
+import io.opentelemetry.javaagent.instrumentation.api.db.normalizer.SqlNormalizer;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public final class CassandraQueryNormalizer {
```

Comment:
this feels cassandra-specific enough, and small enough, to not include in the api